web/src/app/converters/proto-model-converter.ts

```diff
@@ -72,29 +78,39 @@ export function newSurveyToDocument(
   name: string,
   description: string,
   acl: Map<string, Role>,
-  ownerId: string
+  ownerId: string,
+  status: SurveyStatus
 ): DocumentData | Error {
   return toDocumentData(
     new Pb.Survey({
       name,
       description,
       acl: acl.map(role => roleToProtoRole(role)).toObject(),
       ownerId,
+      status,
     })
   );
 }
 
 /**
  * Returns the proto representation of a partial Survey model object.
  */
-export function partialSurveyToDocument(
-  name: string,
-  description?: string
-): DocumentData | Error {
+export function partialSurveyToDocument({
+  name,
+  description,
+  status,
+}: {
+  name?: string;
+  description?: string;
+  status?: SurveyStatus;
+}): DocumentData | Error {
   return toDocumentData(
     new Pb.Survey({
-      name,
+      ...(name && {name}),
```

Comment:
Let's get rid of these extra mappings once we have a single "addOrUpdateSurvey" method managing survey serialization.

Reply:
better in another pr